.github/workflows/nightly-doc-build.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 env:
   DOCUMENTATION_CNAME: 'fluent.docs.pyansys.com'
```

Comment:
Jorge, I guess this CNAME is working with the versioned documentation right?  Previously we would have deployed it to `dev.fluent.docs.pyansys.com` and now it would be hosted at this CNAME instead.

Reply:
That's right, @dnwillia-work. Once the migration to multi-documentation is complete, everything is hosted under the same domain, in this case, "fluent.docs.pyansys.com".